src/v/cluster/tests/partition_properties_stm_test.cc

```diff
@@ -116,10 +116,13 @@ struct partition_properties_stm_fixture : raft::raft_fixture {
                          .records = 50,
                        })
                 .then([&](ss::circular_buffer<model::record_batch> batches) {
+                    return chunked_vector<model::record_batch>(
+                      std::move(batches));
+                })
+                .then([&](chunked_vector<model::record_batch> batches) {
                     return leader_node.raft()
                       ->replicate(
-                        model::make_memory_record_batch_reader(
-                          std::move(batches)),
```

Comment:
clarification: is there something that prevented `chunked_vector<model::record_batch>(std::move(batches));` from being used here directly?